util/stop/stopper_test.go

```diff
@@ -15,13 +15,19 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package util
+package stop
 
 import (
 	"testing"
 	"time"
+
+	"github.com/cockroachdb/cockroach/testutils/fakeflags"
```

Comment:
We can't import `util/log` from `package stop`, but we could import it from `package stop_test`. This would let us avoid the import cycles without introducing `fakeflags`. 

Alternately, this is a case where it might make sense to use environment variables for `--v` and `--vmodule` instead of/in addition to command-line flags. I don't love environment variable for configuration but it might be better in this case than having to go out of our way to ensure that every package defines these flags whether it uses them or not.